modules/t/test-genome-DBs/ontology/ontology/meta.txt

```diff
@@ -55,3 +55,4 @@
 58	patch	patch_109_110_a.sql|schema_version	\N
 59	patch	patch_110_111_a.sql|schema_version	\N
 60	patch	patch_111_112_a.sql|schema_version	\N
+61	patch	patch_112_113_a.sql|schema_version	\N
```

Comment:
Tbh I'm a bit confused about patch updates in the [ensembl-ontology-schema](https://github.com/Ensembl/ensembl-ontology-schema/tree/main/sql) repo. I've only known `a` patches to be added in and it seems logical to add in other patches, but they haven't been added in since Release 97. Shall I have a go adding in b and c patches for 111_112 and see what happens?